src/components/Typography/HX/HX.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render } from '@testing-library/react'
+
+import { Icon } from '../../Icon'
+import { Typography } from '..'
+import { loremIpsum } from '../../../utils/loremIpsum'
+
+const { H1, H2, H3, H4 } = Typography
+
+describe('HX Component', () => {
```

Comment:
We need to add some tests in which we try to get the specific element from the screen in an easy way. ( e.g. screen.getByRole('something', {name: 'something' }) and screen.getyText('something')  )

Reply:
I would also try interactions with tooltip and copyable icon.